hack/tests/create-e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+# upgrade-e2e script is intended to run in CI environmet and will not run localy
+
+export RESOURCEGROUP=$(cat /dev/urandom | tr -dc 'a-z' | fold -w 6 | head -n 1)
+export SOURCE=$1
+
+## configure secrets
+ln -s /usr/local/e2e-secrets/azure/secret secrets
+set +x
+. secrets/secret
+set -x
+
+# start monitoring and record PID
+make monitoring-build
+./monitoring -outputdir=/tmp/artifacts &
+MON_PID=$!
+
+cd /go/src/github.com/openshift/openshift-azure
```

Comment:
would very much like to avoid hard coding paths - can we make this runnable on a dev laptop as well?